pkgs/development/libraries/qt-5/5.4/default.nix

```diff
@@ -152,7 +152,7 @@ let
 
       multimedia = callPackage
         (
-          { qtSubmodule, base, declarative
+          { qtSubmodule, base, declarative, pkgconfig
```

Comment:
Maybe it's better to make this import as optional one? Like `enableGstreamer ? false`. And if it is true then import `pkgconfig`. What do you think?